run.js

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+
+/**
+ * a temporary utility to run local codemods quickly
+ * before we improve stuff upstream.
+ */
+
+const helpMsg = `
+run.js parser=flow codemodsToRun fileOrDirectoryToModify extensions="js,jsx,ts,tsx"
+
+codemodsToRun - paths to codemods which have a codeshift.config.js file.
+                separated by spaces.
+                can also just provide keys from the "shorthands.json" file
+
+examples:
+
+./run.js flow ./community/@pipedrive__convention-ui-react/src/5.0.0/transform.ts ~/projects/some-project-which-uses-cui4/src/
+./run.js flow cui5                                                               ~/projects/some-project-which-uses-cui4/src/ # cui5 is from shorthands.json
+
+`;
+
+const path = require('path');
+const cp = require('child_process');
+
+const peekNextArg = () => process.argv[0];
+const eatNextArg = () => process.argv.shift();
+
+const shouldPrintHelp = () => (
+  (should =
+    !process.argv.length ||
+    ['-h', '--help', '-help', 'help'].includes(peekNextArg())),
+  should && console.log(helpMsg),
+  should
+);
+
+const parseArgv = () => (
+  process.argv.splice(0, 2),
+  shouldPrintHelp() && process.exit(1),
+  {
+    parser: eatNextArg() || 'flow',
+    transformsToRun: parseArrayFromCsv(eatNextArg() || ''),
+    fileOrDirectoryToModify: eatNextArg() || '',
+    extensions: eatNextArg() || 'js,jsx,ts,tsx',
+  }
+);
+
+run();
+
+function run() {
+  let {
+    parser, //
+    transformsToRun,
+    fileOrDirectoryToModify,
+    extensions,
+  } = parseArgv();
+
+  const shorthands = require(path.join(__dirname, './shorthands.json'));
+  console.log({ shorthands });
+
+  transformsToRun = transformsToRun
+    .map(t => {
+      if (t in shorthands) {
+        return resolveTransformsFromShorthand(shorthands[t]);
+      } else {
+        const dir = path.dirname(t);
+        const cmd = `yarn --cwd ${dir} build`;
```

Comment:
wouldn't need this if we implemented it properly in the actual codeshift-cli because then we could just use ts-node for the source files as well, right?

thought, maybe another use case would be someone also having codemods in .ts in their own repo like thought about in https://github.com/CodeshiftCommunity/CodeshiftCommunity/pull/58, so perhaps we'd need to point ts-node dynamically, and not only to own our source files?